test/trento/hosts/projections/host_projector_test.exs

```diff
@@ -251,6 +252,7 @@ defmodule Trento.Hosts.Projections.HostProjectorTest do
         total_memory_mb: Enum.random(1..128),
         socket_count: Enum.random(1..16),
         os_version: Faker.App.version(),
+        arch: Enum.random(Architecture.values()),
```

Comment:
Let's add an assertion to see that this works. Below add the last line:
```
...
assert event.agent_version == host_projection.agent_version
assert event.prometheus_targets == host_projection.arch
assert event.arch == host_projection.prometheus_targets
```
The same for the broadcast:
```
assert_broadcast "host_details_updated",
                     %{
                       agent_version: ^agent_version,
                       ...
                       arch: ^arch
                       provider_data: nil
                     },
                     1000
```